### PR TITLE
Fix the accuracy problem of allclose op when using float64 data type in static mode.

### PR DESCRIPTION
**Pull request by wzzju**
### PR types

Bug fixes

### PR changes

OPs

### Describe

Fix the accuracy problem of allclose op when using float64 data type in static mode.

### CODE REVIEW

File: paddle/fluid/operators/allclose_op.cc [hunk 1]
```diff
@@ -14,8 +14,10 @@
 
 #include "paddle/fluid/operators/allclose_op.h"
 #include <cmath>
+#include <string>
 #include "paddle/fluid/framework/op_registry.h"
 #include "paddle/fluid/framework/op_version_registry.h"
+#include "paddle/fluid/framework/op_version_registry.h"
```
Comment (zhhsplendid) [inline paddle/fluid/operators/allclose_op.cc]: Include same file twice?

Comment (wzzju) [reply paddle/fluid/operators/allclose_op.cc]: Done.

**Review by GaoWei8 — DISMISSED** [review]
Approve for op benchmark CI

**Review by GaoWei8 — APPROVED** [review]

**Review by zhhsplendid — APPROVED** [review]
LGTM

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**